service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php

```diff
@@ -50,6 +46,6 @@ public function getCurrentRoute(): string {
     }
 
     public function getPreviousRoute(): string {
-        return $this->sessionDetails->previousRoute;
+        return $this->sessionDetails->previousRoute ?? 'home';
```

Comment:
i guess it's reasonable to assume home is the default here?

Reply:
Yer, I thought it best to set this to a default of home, with the inconsistency with how the route is populated. I did consider setting it to an empty string.

Reply:
Have had to add another null check on `getCurrentRoute` as this seems to be still being populated with null somehow, despite the earlier check to make sure we have the `routeMatch`

Reply:
I think we're going to need a cypress test for this sharpish. How confident are you that we don't need it for this PR @Raffers ?

Reply:
I picked up the cypress ping test and have done a rough draft, so we can put that in as soon as we've got Nick's cypress work in.